src/router.ts

```diff
@@ -75,11 +74,8 @@ const routes = [
       },
       {
         path: "/access/hardware/ledger",
-        component: AccessLedger,
```

Comment:
??

Reply:
I'm not sure where that came from, that's not mine.